src/device/pf_pdport.h

```diff
@@ -35,13 +35,17 @@ int pf_pdport_init (pnet_t * net);
 /**
  * Reset PDPort configuration data for all ports.
  * Clear DPPort configuration in nvm.
+ * Remove all active diagnostics items related to DPPort.
  *
  * @param net              InOut: The p-net stack instance
  * @return  0  if the operation succeeded.
  *          -1 if not found or an error occurred
  */
 int pf_pdport_reset_all (pnet_t * net);
 
+void pf_pdport_ar_connected (pnet_t * net, pf_ar_t * p_ar);
```

Comment:
I prefer to have the documentation block in the header file (unless it is exported just for unit-testing).

Reply:
I have added documentation for all missing functions in pf_pdport.h. I think you need to check this file again.

Reply:
Done